.repo-metadata.json

```diff
@@ -2,11 +2,12 @@
     "name": "google-api-core",
     "name_pretty": "Google API client core library",
     "client_documentation": "https://googleapis.dev/python/google-api-core/latest",
-    "release_level": "ga",
+    "release_level": "stable",
     "language": "python",
     "library_type": "CORE",
     "repo": "googleapis/python-api-core",
     "distribution_name": "google-api-core",
     "default_version": "",
-    "codeowner_team": "@googleapis/actools-python"
+    "codeowner_team": "@googleapis/actools-python",
+    "api_shortname": "google-api-core"
```

Comment:
This is not a `GAPIC` library, is `api_shortname` still required?

Reply:
Fixed.